src/drafts/MarkdownEditor/_SavedReplies.tsx

```diff
@@ -86,7 +86,14 @@ export const SavedRepliesButton = () => {
 
   return items ? (
     <SelectPanel
-      renderAnchor={props => <ToolbarButton {...props} icon={ReplyIcon} aria-label="Add saved reply (Ctrl + .)" />}
+      renderAnchor={props => (
+        <ToolbarButton
+          {...props}
+          icon={ReplyIcon}
+          aria-label="Add saved reply (Ctrl + .)"
+          aria-labelledby={undefined}
```

Comment:
Is the `aria-labelledby` required here due to parent `props` that's being spread or for a type-related reason?

Reply:
Yep, since we're spreading props that could have either aria-label or aria-labelled by, we need to be sure to not have both (given the pr I referenced that enforced only 1 of these exist (and only 1)